recipes/qnm/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "qnm" %}
+{% set version = "0.4.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 59d364dacbe5ea7b2b62844aad40f9351d987fe6e3c9c9b0d076574a8033d03f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
```

Comment:
I think this package can be noarch, https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#architecture-independent-packages, so can you please add `noarch: python` in the `build` section here?

Reply:
Thanks, @npavlovikj. Do I also want `--no-deps --ignore-installed` ?

Reply:
Added `noarch: python` and `--no-deps --ignore-installed` to the pip command line